packages/fx-core/resource/package.nls.json

```diff
@@ -71,7 +71,7 @@
   "core.aadManifestMigration.outputMsg": "Congratulations! AAD manifest has been added, and please start to use this file to customize your AAD App registration. We have added aad.template.json, updated your projectSettings.json file and backed up projectSettings.json to .backup folder.",
   "core.aadManifestMigration.SuccessMessage": "Detect that your project is a previous version and auto update your project for new Teams Toolkit version. The following changes have been made:\n1. 'aad.template.json' is created under 'templates/appPackage'. For more information about AAD manifest, please refer %s\n2. '.fx/configs/projectSettings.json' is updated to include authentication related configuration. You can find a backup of old version in .backup folder.\nCongratulations! Your project now is updated and ready to run in latest version of Teams Toolkit.",
   "core.aadManifestMigration.ParsePermissionsFailedWarning": "Cannot parse permissions.json file when migration, and please follow this link to update permissions in aad.template.json file if necessary: https://aka.ms/teamsfx-aad-manifest#customize-requiredresourceaccess",
-  "core.projectVersionChecker.vscodeUseNewVersion":"Your Teams Toolkit extension is too old to support current project, please update the extension to latest version and try to open this project again",
+  "core.projectVersionChecker.vscodeUseNewVersion": "Your Teams Toolkit extension is too old to support current project, please update the extension to latest version and try to open this project again",
```

Comment:
Your Teams Toolkit extension no longer supports the current project, please update to the latest version and reopen this project.

Reply:
Thanks, adopted all your suggestions:)